### ISSUE TITLE
Q: dorado basecaller (simplex) | duplex numbers

### ISSUE THREAD

**Issue by sklages** [nanoporetech/dorado#316]
When I run the same POD5 dataset with 
- (a) dorado basecaller -> sup mode 
- (b) dorado duplex -> sup mode (same model)

I get a higher number of seqs with "duplex":
```
# simplex
samtools view -c SMPL.basecalls-sup.bam
3108454

# duplex
samtools view -cddx:1 SMPL.basecalls-sup.duplex.bam
295264
samtools view -cddx:0 SMPL.basecalls-sup.duplex.bam
3135960

sum -> 3,431,224 seqs
```

I would have expected a lower number of seqs for (b) due to the duplex approach. What do I miss here?

**Comment by vellamike** [comment]
Hi @sklages,

There's a couple of things going on here.

1. You are seeing more overall reads with duplex than simplex - this will always be the case, it is because the duplex pipeline performs simplex basecalling, so it outputs all simplex reads as well as the duplex ones.
2. You are seeing more simplex reads in the duplex pipeline than in the simplex pipeline. This is because read splitting is enabled in the simplex pipeline and has not yet been enabled in simplex. It is something we will fix soon.

**Comment by sklages** [comment]
@vellamike - thanks for clarification.

1. I thought that simplex reads would not be written when used in duplex formation.  
So I expected `reads(simplex) + 2x reads(duplex) == reads(total)`. 
How do I identify/remove the simplex reads which are part of a duplex sequence?
2. hmm, I am a bit unsure now which way to go for HQ data, sup-simplex or sup-duplex? Or a combination, duplex data plus simplex data from sup-simplex (those reads not used in duplex formation)?

**Comment by tijyojwad** [comment]
Hi @sklages - I'm closing this issue since there's an overlapping discussion in #327 ?

To answer your question `which way to go for HQ data, sup-simplex or sup-duplex? Or a combination` , I think the best way forward would be to use records with tags `dx:0` (unpaired simplex) and `dx:1` (duplex) as released on v0.3.3 onwards
